src/main/java/ch/njol/skript/registrations/Converters.java

```diff
@@ -287,9 +287,10 @@ public static boolean converterExists(final Class<?> from, final Class<?>... to)
 	 * @return the converter or null if none exist
 	 */
 	@Nullable
-	public static <F, T> Converter<? super F, ? extends T> getConverter(final Class<F> from, final Class<T> to) {
+	@SuppressWarnings("unchecked")
+	public static <F, T> Converter<F, T> getConverter(final Class<F> from, final Class<T> to) {
 		ConverterInfo<? super F, ? extends T> info = getConverterInfo(from, to);
-		return info != null ? info.converter : null;
+		return info != null ? (Converter<F, T>) info.converter : null;
```

Comment:
Why the changes to this method?

Reply:
I originally did this to prevent having to cast some stuff, but I am going to revert it

I am working on a Converters cleanup as well, so any redundant casts for Comparators can be removed in the future if needed